include/classes/Utility.class.php

```diff
@@ -317,10 +317,9 @@ public static function sanitizeIntArray($int_array) {
 		if (!is_array($int_array)) return false;
 		
 		$sanitized_array = array();
-		foreach ($int_array as $i => $value) {
-			$sanitized_array[$i] = intval($value);
-		}
-		return $sanitized_array;
+	
+		return $sanitized_array = array_filter($int_array, 'is_numeric');

```

Comment:
`$sanitized_array = ` can be removed. should be `return array_filter($int_array, 'is_numeric');`.